app/selenium_ui/confluence/pages/selectors.py

```diff
@@ -0,0 +1,82 @@
+from selenium.webdriver.common.by import By
+from util.conf import CONFLUENCE_SETTINGS
+
+
+class UrlManager:
+
+    def __init__(self, page_id=None):
+        self.host = CONFLUENCE_SETTINGS.server_url
+        self.login_params = '/login.action'
+        self.page_params = f"/pages/viewpage.action?pageId={page_id}"
+        self.dashboard_params = '/dashboard.action#all-updates'
+        self.edit_page_params = f'/pages/editpage.action?pageId={page_id}'
+        self.logout_params = "/logout.action"
+
+    def login_url(self):
+        return f"{self.host}{self.login_params}"
+
+    def dashboard_url(self):
+        return f"{self.host}{self.dashboard_params}"
+
+    def page_url(self):
+        return f"{self.host}{self.page_params}"
+
+    def edit_page_url(self):
+        return f"{self.host}{self.edit_page_params}"
+
+    def logout_url(self):
+        return f"{self.host}{self.logout_params}"
+
+
+class PopupLocators:
+    default_popup = '.button-panel-button .set-timezone-button'
+    popup_1 = '.aui-button aui-button-link .skip-onboarding'
```

Comment:
the same as for Jira - let's come up with clear naming for popups selectors.
popup_timezone, popup_skip_onboarding etc

Reply:
Fixed for Confluence